packages/ai/api-report.api.md

```diff
@@ -135,6 +141,28 @@ export interface TLAiUpdateShapeChange<T extends TLShape = TLShape> {
     type: 'updateShape';
 }
 
+// @public (undocumented)
+export interface TldrawAi {
+    // (undocumented)
+    cancel(): void;
+    // (undocumented)
+    prompt(message: TldrawAiPromptOptions): {
+        cancel(): void;
+        promise: Promise<void>;
+    };
+    // (undocumented)
+    repeat(): {
+        cancel: (() => void) | null;
+        promise: Promise<void>;
+    };
+}
```

Comment:
These do have docs but they're in the wrong place so they're not getting picked up

Reply:
Also: I wasnt sure why this wasn't already exported.

Sidenote: `repeat` seems to be for debugging? Should it be internal?